### ISSUE TITLE
Button Remapping, and Joystick Deadzone Change.

### ISSUE THREAD

**Issue by roxas10** [wiredopposite/OGX-Mini#39]
**Is your feature request related to a problem? Please describe.**
I would love to change deadzone on joysticks because I use this adapter with PS3 and Xbox360 Wheels, and in some games I can't Drive Correctly cause has a giant deadzone.
I also need to remap some buttons so I can change the shift buttons.

**Describe the solution you'd like**

**Additional context**

**Comment by wiredopposite** [comment]
A webapp for remapping and adjusting deadzones is coming in the next update. I've rewritten a lot of the firmware, which has taken a while, but it should be pretty soon, within the next week or two

**Comment by mastercheff** [comment]
Also, some sort of Anti-DZ would be greatly appreciated , since most games default to ~25% , Monkey Ball DX is barely playable compared to gamecube port.

![image](https://github.com/user-attachments/assets/1a328309-61ba-4f62-9adc-4d7e104856a8)

**Comment by Invictaz** [comment]
Button remap is in the newest release v1.0 alpha

**Comment by Invictaz** [comment]
Yes but deadzone and anti deadzone is missing so I'm not sure why this is closed?

**Comment by mastercheff** [comment]
deadzone is implemented now We can add more DZ , reducing it would be most beneficial in most games. 2/3rds of the request are completed, and anti DZ is on the roadmap as I see it in the readme, We're cool

**Comment by wiredopposite** [comment]
I'm not ignoring this, I've added it to the roadmap and want to keep the issues page from getting cluttered with too many things that aren't actual problems with the firmware, it makes it harder to parse through. Also the original issue wasn't about anti-dz.
